src/wavm/CMakeLists.txt

```diff
@@ -24,6 +25,7 @@ set(LIB_FILES
         mpi.cpp
         network.cpp
         openmp.cpp
+        PlatformThreadPool.cpp
```

Comment:
Can you rename `PlatformThreadPool` to `OMPThreadPool` as per the comment above re. it being OMP-specific?